docs/architecture/cloud-native/azure-data-storage.md

```diff
@@ -13,7 +13,7 @@ You could just provision an Azure virtual machine and install your database of c
 
 Instead, a fully managed [Database as a Service (DBaaS)](https://www.stratoscale.com/blog/dbaas/what-is-database-as-a-service/) is a better option. You get many built-in features while the hosting, maintenance, and licensing are managed by Microsoft. Azure features different kinds of fully managed data storage options, each with specific benefits. They all support just-in-time capacity and a pay-as-you-go model.
 
-We'll next look at DBaaS options available in Azure. You'll see how Microsoft continues is commitment to keeping Azure an “open platform,” offering managed support for many open-source relational and NoSQL databases and making key contributions to the various open-source foundations as an active member.
+We'll next look at DBaaS options available in Azure. You'll see how Microsoft continues is commitment to keeping Azure an "open platform", offering managed support for many open-source relational and NoSQL databases and making key contributions to the various open-source foundations as an active member.
```

Comment:
According to the current guidelines, commas and periods should be inside quotation marks:
https://docs.microsoft.com/en-us/style-guide/punctuation/quotation-marks

Reply:
@pkulikov, Does this contradict with #16513?

Reply:
@Youssef1313 yes, changes made in #16513 look to contradict the guidelines.

As for #16513 though, I wouldn't use quotation marks there at all. I guess those quoted words would be localized, but they shouldn't as they are parts of members' names which are always in English. I would use backticks there.

Reply:
I agree that they shouldn't be localized, either using backticks or the no-loc metadata.
Regarding the style-guide, I'll wait for @mairaw or @gewarren.
What I understood from it is to usually put the punctuation outside it with the noted exception:

> **Exception** If punctuation is part of the quoted material, place it inside the quotation marks.

Reply:
The referenced page mentions commas and periods separately; also, the examples there put both comma and period inside the quotation marks. 

As for the exception, I understand it refers to the question and exclamation marks, for example.

Reply:
Yes, according to the Microsoft style guide and the Chicago Manual of Style, periods and commas should appear inside the closing quotation mark. This is actually news to me, and I don't like this style :). The Oxford or British style is to always put punctuation outside the closing quotation mark, except if it's part of the quoted material.

Reply:
I've seen that before but I also dislike that style. 😄

Reply:
@mairaw, Yes. The style looks really strange to me. But if it's required I'll do that.